### ISSUE TITLE
Proably mistyping

### ISSUE THREAD

**Issue by brada4** [xianyi/OpenBLAS#2322]
It masks out the bit seeked?
```
dynamic.c: In function ‘support_avx512’:
dynamic.c:332:21: warning: bitwise comparison always evaluates to true [-Wtautological-compare]
  332 |   if((ebx & (1<<7)) != 1){
```

**Comment by martin-frbg** [comment]
Err yes, looks like I did a mindless inversion of the support_avx2() test a few lines above - but 10000000 & 10000000 will still be 128 (10000000) rather than 1. Thanks, fixed in f95989c

**Comment by brada4** [comment]
Thank you.
